doc/glossary.rst

```diff
@@ -119,9 +119,24 @@ general neuroimaging concepts. If you think a term is missing, please consider
         object class, and :ref:`tut-evoked-class` for a narrative overview.
 
     fiducial point
-        There are three fiducial (a.k.a. cardinal) points: the left
-        preauricular point (LPA), the right preauricular point (RPA)
-        and the nasion.
+        Generally speaking, fiducials are objects which are placed in the field of
+        view of an imaging system and which then appear in the produced image.
+        The image of a given fiducial object (or fiducial point) can then be used
+        as a point of reference in that image.
+        In the case of neuroimaging, fiducials are often placed on well-known
+        anatomical landmarks on the body of a study participant such as the nasion
+        (NAS), or left and right preauricular points (LPA and RPA), also known as
+        cardinal points.
+        They are then used to facilitate the localization of sensors
+        and for their co-registration with other geometric data such as the
+        participant's own T1 weighted magnetic resonance head image,
+        a T1 weighted template head image, or a spherical head model.
+
+        Note that due to the common use of placing fiducial objects on anatomical
+        landmarks, the terms "fiducial" and "anatomical landmark" are often
+        (erroneously) used synonymously. However, fiducials can be placed
+        anywhere in an image, and do not have to coincide with an anatomical
+        landmark.
```

Comment:
This is a great addition.  It feels a little verbose to me.  Can we shorten it without losing the important added info? Here's an attempt that makes it a little shorter:

```suggestion
        Fiducials are objects placed in the field of view of an imaging system
        to act as a known spatial reference location that is easy to localize.
        In neuroimaging, fiducials are often placed on anatomical landmarks
        such as the nasion (NAS) or left/right preauricular points (LPA and
        RPA).

        These known reference locations are used define a coordinate system 
        used for localization of sensors (hence NAS, LPA and RPA are often
        called "cardinal points" because they define the cardinal directions of
        the "head" coordinate system). The cardinal points are also useful when
        co-registering measurements in different coordinate systems (such as
        aligning EEG sensor locations to an MRI of the subject's head).

        Due to the common neuroimaging practice of placing fiducial objects on
        anatomical landmarks, the terms "fiducial", "anatomical landmark" and
        "cardinal point" are often (erroneously) used interchangably.
```

Reply:
This is great, I thik there is only a `"to"` missing:

> These known reference locations are used define a coordinate system 
        used for localization of sensors 
        
I'll edit your suggestion to add that, and the batch-accept both of your suggestions